build_dict.pl

```diff
@@ -42,7 +42,7 @@
 		raw_simp => $simp,
 		py => [split ' ', $py],
 		raw_py => lc($py),
-		defs => [split '/', $defs],
+		defs => [split '/', $defs =~ s/&/&amp;/rg],
```

Comment:
Some similar escaping happens down below around line 142. Can you move this down there so the XML escaping is all in one place?